### ISSUE TITLE
模型根本不对，请作者仔细看下哦

### ISSUE THREAD

**Issue by JavaLeb** [zhaoyingjun/chatbot#86]
def train_step(inp, targ, targ_lang,enc_hidden):
  loss = 0
  with tf.GradientTape() as tape:
    enc_output, enc_hidden = encoder(inp, enc_hidden)
    dec_hidden = enc_hidden
    dec_input = tf.expand_dims([targ_lang.word_index['start']] * BATCH_SIZE, 1)
    for t in range(1, targ.shape[1]):
      predictions, dec_hidden, _ = decoder(dec_input, dec_hidden, enc_output)
dec_input维度是(128,1)，传给了decoder，decoder的第一个参数是词典大小。
根本就不对

**Comment by wenChingzhou** [comment]
何解呢
